### ISSUE TITLE
Move style-dependent hyper parameters to replication for johnson_alahi_li_2016

### ISSUE THREAD

**Issue by pmeier** [pmeier/pystiche_papers#217]
Since the beginning (#10) we included hyper-parameters in  `johnson_alahi_li_2016`, which are depended on a specific style. The origin of these hyper-parameters was clarified in #143. For example:

https://github.com/pmeier/pystiche_papers/blob/9f2acb82191353c55370326a5f4a91cd7528d969/pystiche_papers/johnson_alahi_li_2016/_loss.py#L21-L32

IMO, we shouldn't include this in the reference implementation, but rather in the replication scripts. This is similar to what we did in #165 for `li_wand_2016`.

Cc @jbueltemeier

**Comment by jbueltemeier** [comment]
I agree with you that this should be changed to make the implementation more transparent.

As a small side note, since I just saw that if this is changed in [L41](https://github.com/pmeier/pystiche_papers/blob/9f2acb82191353c55370326a5f4a91cd7528d969/pystiche_papers/johnson_alahi_li_2016/_loss.py#L41) the typo should be corrected. ("style" should be "content" here).
